### DISCUSSION TITLE
Any tips to unlock the Piano?

### DISCUSSION THREAD

**Discussion by 3nvu** [Monika-After-Story/MonikaModDev#10262] [QA - tech support + general] [answered]
I've been reading that someone said that I must have like 150 hours spent, then lv 12, and so many more. So I need to know what's the true requirements to unlock the Piano.

**Comment by Wingdinggaster656** [comment]
Mostly, time.

**Comment by 3nvu** [comment, marked as answer]
IDK why but I got it's at 3 days 12 hour, I think the requirements to get the piano is to get Monika play "Your Reality" for you and then you can access to the piano

**Comment by William-Delancey** [comment]
I'm having this same issue,
could you help me too?

**Reply by 3nvu** [reply]
For my "Opinion" when Monika is asking you of she doing something you need to accept it, if she playing your reality then you could have the Piano, IDK if it's work or not that's just what I remember.
